### ISSUE TITLE
Virus warning on executable

### ISSUE THREAD

**Issue by tigrrrlily** [ArtemkaKun/wmr-usb-controller#2]
The executable in 4.0.1 results in a Bitdefender block and quarantine. Bitdefender is mum on which heuristic was triggered, but VirusTotal picks out IsDebuggerPresent and GetTickCount. That, and disabling error reporting?

Admittedly I don't know what I'm doing, but is there any reason you're checking for a debugger? Or has the exe picked up a hitchhiker?

**Comment by ArtemkaKun** [comment]
Hi,

Very interesting. I will try to reproduce the error by myself and will report the results.

**Comment by ArtemkaKun** [comment]
Seems VirusTotal didn't find any "virus" here. Also, I'm not sure if the BitDefenderTheta is the antivirus you mean, but seems it didn't find anything as well.

![image](https://user-images.githubusercontent.com/36485221/108817303-f718e180-75b7-11eb-909f-9315562d65e0.png)

**Comment by ArtemkaKun** [comment]
@tigrrrlily, can you please provide the way where I can find these messages from the VirusTotal?

> VirusTotal picks out IsDebuggerPresent and GetTickCount

**Comment by tigrrrlily** [comment]
Those are the heuristics that the "behavior" tab in VirusTotal reported as
problematic. That analysis puts the tool into sandboxes and lets it run,
and monitors for behaviors that are associated with malware. It's not
signature-based, so no, there's no actual virus involved, most likely.
However, some tools like Bitdefender's "advanced threat defense" (on by
default) will let the software run for a few seconds, keep a running score,
and when their score trips over to "this is doing stuff that malware might
do" terminate the process and move the executable into quarantine. This is
why I fed it to VirusTotal, to try to guess what BD might be having a
problem with.

There's probably nothing wrong with your software, most likely. But if you
let the behavior analysis in VT run, it will tell you the calls you're
using or the behaviors that might make tools think there is. That's all.
Just trying to help out so your users don't get spurious warnings.

And, also, kindof looking for confirmation that you *are* in fact doing all
those things on purpose. Just to make extra-sure.

**Comment by ArtemkaKun** [comment]
Ok, I found it. Thank you for the info.

So, what I got - I rebuild the project one more time in the release mode and still have this flag "detect-debug-environment". I'm not sure if this is the problem with the C#, .NET5 or with the Rider. 

Anyway, I also found this [article](https://www.csoonline.com/article/3216765/heres-why-the-scanners-on-virustotal-flagged-hello-world-as-harmful.html), when the "Hello World" program detected as Trojan by antiviruses.

So, I'm not sure if this is an issue at all :) I can sure you that the program doesn't contain any harmful code that wants to steal your OnlyFans subscription (joke).

Thank you for info, that was useful experience.
